frontend/src/main.js

```diff
@@ -7,7 +7,6 @@ import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap-vue/dist/bootstrap-vue.css';
 
```

Comment:
is there a need for bootstrap-vue anymore?

Reply:
Will be handled after

Reply:
The cards are still written in bootstrap-vue